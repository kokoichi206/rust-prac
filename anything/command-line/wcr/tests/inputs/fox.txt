The  quick brown fox^Ijumps over   the lazy dog.
