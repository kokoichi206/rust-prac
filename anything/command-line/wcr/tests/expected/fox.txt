	1	8	49 tests/inputs/fox.txt
